rust-mode.el

```diff
@@ -170,7 +170,18 @@ function or trait.  When nil, where will be aligned with fn or trait."
   "Face for the question mark operator."
   :group 'rust-mode)
 
+(defface rust-builtin-formatting-macro-face
+  '((t :inherit font-lock-builtin-face))
+  "Face for builtin formatting macros \(print! &c.\)."
+  :group 'rust-mode)
+
+(defface rust-string-interpolation-face
+  '((t :slant italic :inherit font-lock-string-face))
```

Comment:
I figure it’s better to keep it conservative by default and let users customize it if they want something dramatic. Having said that, the italics stand out nicely for me. I guess it’s different from font to font.